test/PoolFactory.js

```diff
@@ -8,13 +8,37 @@ describe("PoolFactory contract", function () {
 
   beforeEach(async function () {
     poolFactoryContract = await ethers.getContractFactory("PoolFactory");
-    [owner] = await ethers.getSigners();
+    [owner, addy1, addy2] = await ethers.getSigners();
 
     poolFactoryToken = await poolFactoryContract.deploy();
     await poolFactoryToken.deployed();
-
-    await poolFactoryToken.deployed();
   });
 
-  // TODO: Add tests
+  
+  describe("Pool dynamics", function () {
+    // `it` is another Mocha function. This is the one you use to define your
+    // tests. It receives the test name, and a callback function.
+
+    // If the callback function is async, Mocha will `await` it.
+    it("Should create a pool", async function () {
+      //create a pool
+      poolRef = await poolFactoryToken.createPool("tadPool", [addy1.address]);
```

Comment:
Love the idea but let's use real names :) 
```suggestion
      poolRef = await poolFactoryToken.createPool("TestPool1", [addy1.address]);
```